gramps/gui/widgets/monitoredwidgets.py

```diff
@@ -141,14 +139,10 @@ def connect(self, signal, callback, *data):
         self.obj.connect(signal, callback, *data)
 
     def _on_quit(self, obj, event):
-        text = obj.get_text().translate(INVISIBLE).strip()
-        self.set_val(text)
-        obj.set_text(text)
+        self.set_val(obj.get_text().strip())
```

Comment:
The only thing I see for the moment, is we don't see anymore the change after the strip.
I propose to have the obj.set_text(text) available again.

Reply:
Same thing in the on_change method.

Reply:
This problems occurs when copy/past